models/tutorial/index.js

```diff
@@ -82,3 +104,19 @@ function handleClockEvent (events, event, send) {
     })
   }
 }
+
+function getLevelIfExistent (category, index) {
```

Comment:
Could you specify what you mean and give a suggestion? Maybe we can change this after tomorrow if it is a code style issue.